minimesos/src/main/groovy/com/containersol/minimesos/config/ClusterConfig.groovy

```diff
@@ -53,4 +53,14 @@ class ClusterConfig extends GroovyBlock {
         delegateTo(marathon, cl)
     }
 
+    void setLoggingLevel(String loggingLevel) {
+         if (!loggingLevel.equals("WARNING") && !loggingLevel.equals("INFO") && !loggingLevel.equals("ERROR")) {
+             throw new RuntimeException("Property 'loggingLevel' can only have the values INFO, WARNING or ERROR")
```

Comment:
* WARN is normally used for warnings. I think we should allow both and I would include ERR too
* I would make it case insensitive